lldb/source/API/SBInstructionList.cpp

```diff
@@ -138,7 +141,14 @@ bool SBInstructionList::GetDescription(lldb::SBStream &stream) {
   return GetDescription(stream.ref());
 }
 
-bool SBInstructionList::GetDescription(Stream &sref) {
+bool SBInstructionList::GetDescription(lldb::SBStream &stream,
+                                       lldb::SBExecutionContext &exe_ctx) {
+  LLDB_INSTRUMENT_VA(this, stream);
+  return GetDescription(stream.ref(), &exe_ctx);
+}
+
+bool SBInstructionList::GetDescription(Stream &sref,
+                                       lldb::SBExecutionContext *exe_ctx) {
```

Comment:
This could take the private type (e.g. `ExecutionContext*`).

Reply:
:100: